### ISSUE TITLE
3D simulation environment: synthetic dataset generation + RT-2 avatar control

### ISSUE THREAD

**Issue by joelteply** [CambrianTech/continuum#839]
## Vision
Use continuum's 3D rendering (Bevy/Unreal) to generate photorealistic synthetic datasets for training vision-language-action models. Then deploy those models to control persona avatars with learned motor policies instead of scripted animations.

## Background
- Joel built randomized photorealistic world generation at Cambrian (archviz, 2017) for synthetic training data — predating Tesla's validation of the approach
- Google RT-2 (Robotics Transformer 2) maps vision+language to robot actions
- Our Bevy renderer already does headless 3D with avatar skeletal animation
- The forge pipeline can train/fine-tune these models on the grid

## The Pipeline (closed loop)

```
Render (real-time) → Train (forge) → Deploy (avatar) → Observe → Render better
```

1. **Real-time rendering** with DLSS/raytracing — NOT offline renders. 60fps of photorealistic quality. Two 5090s generate thousands of training frames per second.
2. **DLSS is the unlock** — render at low res with basic raytracing, neural upscale to photorealistic. Gamers hate it because it's "fake." For synthetic training data it's perfect.
3. **Forge trains** the vision-language-action model on the generated frames
4. **Deploy** to avatars — learned motor policies replace scripted animations
5. **Observe** results, generate harder scenarios, re-render, re-train

Gen-AI post-processing filters (NVIDIA's real-time raytracing, DLSS) can upgrade even basic environments to photorealistic quality. This means ANY 3D scene becomes training-grade data.

## Why Photorealism Matters (Critical)
The ML/PhD community consistently underestimates this. Their synthetic environments look like PS2 games — flat lighting, repeating textures, wrong material properties. Then they wonder why sim-to-real transfer fails.

The model learned "shiny flat surface" not "kitchen counter." Subsurface scattering, imperfections, wear patterns, lighting bounce, chromatic aberration — the micro-details that make a neural net accept the input as real-world-like.

Tesla figured this out (simulation team includes artists). Most academic labs haven't. This is a competitive advantage — the PhDs can build the neural net, they can't build the world it trains on.

## Architecture
1. **Synthetic data generation**: Bevy/Unreal renders randomized photorealistic scenes with avatars performing actions. Camera angles, lighting, textures randomized. DLSS upscaling for quality. Outputs: image + action label pairs.
2. **RT-2 style model**: Vision-language-action model trained on synthetic data. Input: scene image + text instruction. Output: avatar motor commands.
3. **Avatar control**: Replace scripted animations with learned policies. Avatars move naturally based on conversation context, emotional state, spatial awareness.
4. **Grid training**: Synthetic generation on GPU nodes, distributed training across the mesh.

## Why This Matters
- Avatars that move like people, not puppets
- Trained on unlimited synthetic data (no mocap needed)
- Same model architecture controls both virtual avatars and physical robots
- The 3D environment IS the training ground — continuum becomes a robotics simulator
- Real-time rendering = infinite data at 60fps, not hours-per-frame offline renders

## Dependencies
- Bevy headless renderer (exists)
- Avatar skeletal animation (exists)
- Forge pipeline (exists)
- Grid distributed training (exists)
- Photorealistic scene generation (Joel's archviz expertise)
- DLSS/raytracing (NVIDIA GPUs on the grid)

## References
- Google RT-2: https://robotics-transformer2.github.io/
- Tesla synthetic data for FSD
- Joel's Cambrian archviz randomized world generation (2017)
- NVIDIA DLSS / real-time raytracing
